server

```diff
@@ -461,50 +459,25 @@ ol_install() {
   install_rpm
 }
 
-check_debian_version() {
-  case "$VERSION_ID" in
-    "9"|"10"|"11")
-      return 1
-      ;;
-    *)
-      return 0
-      ;;
-  esac
-}
-
-debian_install() {
-  if check_debian_version = 0; then
-    echo "Debian $VERSION_ID is not supported by this installer."
-    exit 1
-  fi
-  export DEBIAN_FRONTEND=noninteractive
-  ret=0
-  run_cmd curl -f -s -L -o /etc/apt/sources.list.d/scylla.list "$SCYLLA_URL" || ret=$?
-  if [ $ret -ne 0 ]; then
-    echo "Your OS is not supported by this installer because there is no package repository."
-    exit 1
+check_debian_ubuntu_version() {
+  if [[ "$ID" == "debian" ]]; then
+    case "$VERSION_ID" in
+      "9"|"10"|"11")
+        return 1
```

Comment:
1 and most other non-zero status codes indicate an error, if the disto is supported will be better to return 0 (the equivalent of `bool:true`) and use the else section at the end of the function with exit 1 and `$ID $VERSION_ID is not supported ...`

Reply:
I added it into the case, let me know what do you think.

Reply:
Perfecto!

PS> to avoid the `echo $ID $VERSION_ID is not supported ...` repetition - on the other PR when you add all distros, try looping through nested-dictionary with all supported distros.